inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/visit_recordings/VisitObservationType.java

```diff
@@ -88,6 +89,16 @@ public class VisitObservationType extends TemporalCore<VisitObservationType, Vis
      */
     private String description;
 
+    /**
+     * \brief Flag for if this row is updating in real time.
+     */
+    private Boolean isRealTime;
+
+    /**
+     * \brief Flag for if this row has an associated row in vist_observation.
```

Comment:
I don't think flags would be common to all our users, better to use english
```suggestion
     * \brief Does this ... associated row in vist_observation.
```